src/main/java/com/starkbank/Deposit.java

```diff
@@ -307,6 +307,47 @@ public static Page page(Map<String, Object> params, User user) throws Exception
         }
         return new Page(deposits, page.cursor);
     }
+
+    /**
+     * Update notification Deposit entity
+     * <p>
+     * Update the Deposit by passing id. If the deposit has not yet been paid, you can adjust parameters
+     * such as the requested amount, the due datetime and the expiration. If the deposit has already been
+     * paid, only the amount can be decreased, which will result in a payment reversal.
+     * <p>
+     * Parameters:
+     * @param id        [string]: Deposit unique id. ex: "5656565656565656"
+     * @param patchData map of parameters to patch
+     *                  amount [string]: If the Deposit hasn't been paid yet, you may update its amount by passing the desired amount integer
+     * <p>
+     * Return:
+     * @return Deposit object with updated attributes
+     * @throws Exception error in the request
```

Comment:
Essa docstring não faz sentido para o Deposit. Entendo que foi inspirado na Invoice, mas são sistemas diferentes.

Update the Deposit by passing id. Only the amount can be decreased, which will result in a payment reversal.

@param patchData map of parameters to patch
     *                  amount [string]: The new amount of the DynamicBrcode. If the amount = 0 the Deposit will be fully reversed